modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,315 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-running-dpdk-rootless-tap_{context}"]
+= Using the TAP CNI to run a rootless DPDK workload with kernel access
+
+In OpenShift Container Platform 4.14 and greater, DPDK pods can run as a normal user by using the TAP plugin to provide a bridged network connection between the container and the host operating system.
```

Comment:
This is not correct. It's a bit confusing. First, I would start giving a short description of the solution used here. It could be something like "DPDK applications can use virtio-user as exception path to inject some type of packets (i.e. log messages) into the kernel for processing there". We could add a reference to https://doc.dpdk.org/guides/howto/virtio_user_as_exception_path.html. Then, we can mention something like "This solution requires the vhost-net device to be mounted in the pod which can be done by enabling the needVhostNet parameter of the SriovNetworkNodePolicy object". Then, for a rootless pod, we can write something like "In OpenShift Container Platform 4.14 and greater, non-priviledged pods can run this type of DPDK application in conjuntion with the tap CNI plugin." This is what it is new.